plugins/remark-snackplayer/package.json

```diff
@@ -21,11 +21,10 @@
   "dependencies": {
     "dedent": "^0.7.0",
     "object.fromentries": "^2.0.3",
-    "unist-builder": "^2.0.3",
     "unist-util-visit-parents": "^3.1.1"
   },
   "devDependencies": {
-    "remark": "^12.0.1",
+    "remark": "^11",
```

Comment:
why this has been downgraded? 🤔

Reply:
Hmm, can't remember and looks suspicious. Reverting.